### PR TITLE
[TR] Add Rules, [EN/FR] remove duplicate explanation

### PR DESCRIPTION
**Pull request by zeusminus**
Add Turkish translation of `Rules` article.

### CODE REVIEW

**Comment by zeusminus** [comment]
Added Turkish translation of `FAQ` article.

**Comment by zeusminus** [comment]
Translation of `Rules` article was reviewed by native speakers beforehand. It is completely ready for merging.

**Comment by cl8n** [comment]
this isn't up-to-date with the latest english version, i'm noticing the beatmap submission rules and off-topic rules are different but there might be more

File: wiki/Rules/tr.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+- Resmi ev sahipliği yapılan turnuvalarda oynama veya tur atlama hakkınızın elinizden alınması
+
+Susturulma cezaları ve diğer kural ihlallerin süresi **otomatik olarak** her seferinde katlanarak artar. Moderatörler aldığınız cezanın süresini control etmez – kuralları sürekli çiğneyenler kendilerini çok daha uzun süreler boyunca oyuna giremez bir konumda bulurlar.
+
+Birtakım kural ihlalleri, özellikle ağır bir tane olması halinde, belirli bir minimum süre bulundururlar (genellikle 48 saat veya daha fazla), lakin bu cezalar hemen hemen yalnızca çok ağır suçlara mahsustur; müstehcen içerik paylaşmak, başkalarına devamlı zorbalık yapmak gibi.
+
+Eğer birçok kez ceza almanıza rağmen yine de kuralları çiğnemeye devam ederseniz, hesabınız süresiz ve geri alınamaz bir şekilde kısıtlandırılabilir.
+
+Kısıtlı hesaplar çevrimiçi oynayamazlar, skor kaydedemezler, diğer kullanıcılarla sohbet edemezler, ve tüm bu niyetler ve amaçlar doğrultusunda topluluk ile iletişim kurmaktan men edilirler. Topluluk kurallarını çiğnemekten kısıtlandırılmış hesapların cezası ancak 3 ay geçtikten sonra oyun yetkililerine başvurulması halinde kaldırılabilir.
```
Comment (cl8n) [inline wiki/Rules/tr.md]: needs the link to account restrictions, see english

**Comment by cl8n** [comment]
looks like all the rules mean the same thing at least so this is good for now

**Pull request merged**